app/src/main/java/com/rentminder/MainActivity.kt

```diff
@@ -118,10 +105,57 @@ fun TopToolBar() {
     }
 }
 
-//Bill input boxes, buttons, and text fields
+/**
+ * 2. Column (Calendar Display Name)
+ * Displays current calendar in a month-only format
+ */
+@Composable
+fun CalendarNameColumn()
+{
+    //Getting current month for Main Menu
+    val calendar: Calendar = Calendar.getInstance()
+    val monthDate = SimpleDateFormat("MMMM")
+    val monthName: String = monthDate.format(calendar.time)
+
+    Row(modifier = Modifier
+        .padding(
+            top = 5.dp,
+            bottom = 10.dp
+        )
+        .fillMaxWidth(),
+        horizontalArrangement = Arrangement.Center
+    ) {
+        Text(
+            text = monthName,
+            fontSize = 28.sp,
+            fontWeight = FontWeight.Bold
+        )
+    }
+}
+
+/**
+ * 3. Column (All type of bills)
+ * Bill input boxes, buttons, and text fields.
+ *
+ * 1. Row (Rent)
+ * Properties:
+ *
+ * 2. Row (Electric)
+ * Properties: 
+ *
+ * 3. Row (Water)
+ * Properties:
+ *
+ * 4. Row (Wifi)
+ * Properties:
+ *
+ * 5. Row (Miscellaneous)
+ * Properties:
+ *
+ */
```

Comment:
This follows the previous comment about providing accurate and well organized Kotlin documentation. Here is where I implemented a proper comment that illustrates how RentBillColumn constructs the UI. You can add specific properties to which you would like to modify and track with this documentation.

Reply:
Careful to not be toooooo detailed in KDoc, as you'll have to keep this in sync with the source code itself.  May be better to let the source code do the talking.